### PR TITLE
Add com.github.geigi.cozy

### PR DESCRIPTION
**Pull request by geigi**
Hey,
I've already tested the recipe locally on my machine and it was working nice ;)

### CODE REVIEW

File: com.github.geigi.cozy.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.geigi.cozy",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-music",
```
Comment (nedrichards) [inline com.github.geigi.cozy.json]: I meant to think about this before, but you might be able to expose this as :ro if you don't need to write anything back, which would improve the sandboxing.

**Comment by nedrichards** [comment]
bot, build com.github.geigi.cozy

**Comment by nedrichards** [comment]
bot, build com.github.geigi.cozy

File: com.github.geigi.cozy.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          ],
+          "cleanup": [ "*.la", "/share/gtk-doc" ],
+          "sources": [{
+            "type": "archive",
+            "url": "https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.12.3.tar.xz",
+            "sha256": "015ef8cab6f7fb87c8fb42642486423eff3b6e6a6bccdcd6a189f436a3619650"
+          }]
+        },
+        {
+          "name": "gst-plugins-bad",
```
Comment (nedrichards) [inline com.github.geigi.cozy.json]: Do you need all the plugins here? Overall we prefer to [reduce the scope](https://github.com/flathub/com.github.wwmm.pulseeffects/blob/30ad485fedb0c1d73936911d66f4a078277e0219/com.github.wwmm.pulseeffects.json#L114) as a) less unnecessary storage on the users machine and b) you're affected by fewer potential security issues.

Comment (geigi) [reply com.github.geigi.cozy.json]: Oh somehow this got lost... I've just tested it with both libraries separately and `gst-libav` is enough to support the audio codecs I need (AAC and ALAC for this plugin)

**Comment by nedrichards** [comment]
bot, build com.github.geigi.cozy

File: com.github.geigi.cozy.json (hunk 1 above)
Comment (geigi) [reply com.github.geigi.cozy.json]: Oh I think I didn't understand the GitHub review process yet, or did you see my comment here already?
The user can add new audio files by drag & drop: Those will be copied to the selected directory and scanned by cozy. So I think we need rw access here?

Comment (nedrichards) [reply com.github.geigi.cozy.json]: Yeah, lgtm

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.geigi.cozy

**Pull request closed without merging**